### PR TITLE
README.md for changes 

### PR DESCRIPTION
**Pull request by tomjenningss**
This is with the changes from https://github.com/OpenLiberty/quick-labs/pull/13

### CODE REVIEW

File: instructions/guide-reactive-messaging/README.md [hunk 1]
```diff
@@ -0,0 +1,588 @@
...
+</project>
+```
+
+In order to develop the MicroProfile Reactive Messaging application using Maven, add the **microprofile-reactive-messaging-api**, **kafka-clients**, and **rxjava** dependencies to the **pom.xml**.
+
+**microprofile-reactive-messaging-api** dependency is needed to enable the use of MicroProfile Reactive Messaging API. **kafka-clients** dependency is added since the application needs a Kafka client to connect to the Kafka broker. **rxjava** dependency is used for creating events at regular intervals.
+
+# Building the Consumer in the inventory microservice
+
+The **inventory** microservice consumes the events produced by the system microservice and stores the information about the CPU usage that runs on different systems.
```
Comment (GraceJansen) [inline instructions/guide-reactive-messaging/README.md:297]: should system in system microservice have two stars either side?

File: instructions/guide-reactive-messaging/README.md [hunk 2]
```diff
@@ -0,0 +1,588 @@
...
+
+# Configuring the MicroProfile Reactive Messaging connectors
+
+MicroProfile Reactive Messaging uses connectors to send and receive messages from different systems. In order to configure the MicroProfile Reactive Messaging connectors, you need to format the configuration keys as follows :
+
+1. mp.messaging.connector.liberty-kafka.<property name>: Applies to all channels using the liberty-kafka connector
+
+2. mp.messaging.[incoming|outgoing].<channel name>.<property name> : Applies to a particular channel
+
+3. Each channel which is to be connected using a connector must have the connector property defined to say which **connector** to use
```
Comment (GraceJansen) [inline instructions/guide-reactive-messaging/README.md:407]: should connector be in bold here? it isn't anywhere else

File: instructions/guide-reactive-messaging/README.md [hunk 3]
```diff
@@ -0,0 +1,588 @@
...
+
+Start the application 
+
+`./scripts/startContainers.sh`
+
+# Test the application
+
+Once the application is up and running, you can access the application by making a **@GET** request to the **inventory** endpoint.
+
+To access the **inventory** microservice, use the **inventory/systems** URL, and you see the CPU systemLoad property for all the systems.
```
Comment (GraceJansen) [inline instructions/guide-reactive-messaging/README.md:559]: should systemload be in bold here? it is below.

File: instructions/guide-reactive-messaging/README.md [hunk 4]
```diff
@@ -0,0 +1,588 @@
...
+
+### What is Kafka?
+Apache Kafka is a stream-processing platform that manages communication in distributed systems. Communication is message-oriented, and follows the publish-subscribe model. Kafka allows for real-time stream processing and distributed, replicated storage of streams and messages. A Kafka producer is a client or a program, which produces the message and pushes them to a topic. A Kafka consumer is a client or a program, which consumes the published messages from a topic.
+
+The application that you will be working with consists of two microservices, **system** and **inventory**. The system microservice sends the average **system** load data to the **inventory** microservice every 15 seconds. The **inventory** microservice keeps an updated list of all the **system** hostnames and their CPU data.
```
[the remark this answers is not in the document]
Comment (tomjenningss) [reply instructions/guide-reactive-messaging/README.md:25]: I think it is good @GraceJansen . I will add it in, I know that @lauracowen is making editorial changes to the content as well :)

File: instructions/guide-reactive-messaging/README.md [hunk 5]
```diff
@@ -0,0 +1,588 @@
...
+
+### What is reactive programming?
+Reactive programming is a method for writing code based on react‐ing to changes. In technical terms, this is a paradigm in which declarative code is used to construct asynchronous processing pipelines. Translated, this is essentially the same process our minds perform when we try to multitask. Rather than true parallel tasking, we actually switch tasks and split those tasks during their duration. This enables us to use our time efficiently instead of having to wait for the previous task to complete. This is exactly what reactive programming was created to do. It is an event-based model in which data is pushed to a consumer as it becomes available, turning it into an asynchronous sequence of events. Reactive programming is a useful implementation technique for managing internal logic and data flow transformation locally within components (intercomponents), through asynchronous and non-blocking execution.
```
[the remark this answers is not in the document]
Comment (tomjenningss) [reply instructions/guide-reactive-messaging/README.md:17]: @GraceJansen not usually. Its been taken out :)

File: instructions/guide-reactive-messaging/README.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (tomjenningss) [reply instructions/guide-reactive-messaging/README.md:25]: I think it is good @GraceJansen . I like it. I know that @lauracowen is making editorial changes to the content as well so @lauracowen this might be useful for you as well.

[the remark this answers is not in the document]
Comment (GraceJansen) [reply instructions/guide-reactive-messaging/README.md:25]: Not sure we need an additional sentence for this one, just a sebsection within the old sentence. So maybe remove the sentence you've added and add in the reference to the topic in the second sentence of this paragraph.

File: instructions/guide-reactive-messaging/README.md [hunk 6]
```diff
@@ -0,0 +1,592 @@
...
+
+### Tearing down the environment 
+
+Finally, use the following script to stop the application:
+
+`./scripts/stopContainers.sh`
+
+# Well done
+
+You have experienced building reactive microservices using MicroProfile Reactive Messaging, and also sending messages between the microservices using Apache Kafka.
```
Comment (GraceJansen) [inline instructions/guide-reactive-messaging/README.md:592]: Think we can maybe rephrase it. Maybe something like: "Congratulations, you now have experienced building a reactive microservice-based application using MicroProfile Reactive messaging, and experienced sending and receiving messaging between microservices within this application using Apache Kafka."

File: instructions/guide-reactive-messaging/README.md [hunk 7]
```diff
@@ -0,0 +1,560 @@
...
+}
+```
+
+The **SystemService** class contains a **Publisher** method that is called **sendSystemLoad()**, which calculates and returns the average system load. The **@Outgoing** annotation on the **sendSystemLoad()** method indicates that the method publishes its calculation as a message on a topic in the Kafka messaging system. The **Flowable.interval()** method from **rxJava** is used to set the frequency of how often the system service publishes the calculation to the event stream.
+
+The messages are transported between the service and the Kafka messaging system through a channel called **systemLoad**. The name of the channel to use is set in the **@Outgoing("systemLoad")** annotation. Later in the guide, you will configure the service so that any messages sent by the **system** service through the **systemLoad** channel are published on a topic called **systemLoadTopic**. 
+
+# Creating the consumer in the inventory microservice
+
+The **inventory** microservice records in its inventory the average system load information that it received from potentially multiple instances of the system service.
```
Comment (GraceJansen) [inline instructions/guide-reactive-messaging/README.md:105]: should system service be in bold here, as it is above, or not?

Comment (tomjenningss) [reply instructions/guide-reactive-messaging/README.md:105]: @GraceJansen I have just checked, its not bold on the final lab. This is because its talking about a system load ie the amount of computational work, not the `system` service

Comment (tomjenningss) [reply instructions/guide-reactive-messaging/README.md]: @GraceJansen I have added a highlight to system at the end of the sentence good spot 👍.
